### PR TITLE
Fix karma test to pass in travis-ci

### PR DESCRIPTION
**Pull request by kmkzt**
Fix karma test to pass in travis-ci.
I'd like you to point out any changes.

## Made the following major changes
- node_version changed to 8,10,12
- Changed to not manage yarn.lock, package-lock.json

## refference
https://stackoverflow.com/questions/55674746/travis-sh-0-cant-open-etc-init-d-xvfb
https://docs.travis-ci.com/user/chrome#karma-chrome-launcher
https://github.com/twobin/react-lazyload/pull/284#issue-387525643

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 node_modules/
 coverage/
 .DS_Store
+yarn.lock
```
Comment (ameerthehacker) [inline .gitignore]: can you please remove the lock files from gitignore as it is required for consistent dev setup

**Comment by ameerthehacker** [comment]
@kmkzt thanks for the contribution, this definitely helps!

File: .gitignore (hunk 1 above)
Comment (kmkzt) [reply .gitignore]: Thank you for the review. Fixed.
Do I need to update the lock file?

Comment (ameerthehacker) [reply .gitignore]: No everything is perfect

**Comment by kmkzt** [comment]
@ameerthehacker Thanks!

**Pull request merged**